source/d_qgis_plugin.rst

```diff
@@ -5,40 +5,39 @@ Modeller Interface
 
 Introduction
 --------------
-The Modeller Interface is the interface to use on your laptop to build and analyse 3Di models and results locally. To do this, no internet connection is required. It is also the interface to interact with the API and download results from the 3Di calculation servers. It consists of QGIS with a preinstalled plugins: 3Di Toolbox to analyse results, 3Di API Client to start calculations and some third party plugins. The interface has been cleaned compared to a standard QGIS installation to show only relevant buttons for model building analysing. 
+The Modeller Interface (MI) will help you with building 3Di models and analysing results locally. The MI will also assist you in interacting with the 3Di-API and downloading the results from the 3Di calculation servers. The MI is part of QGIS with various pre-installed plugins: the 3Di Toolbox to analyse results, the 3Di API Client to start calculations and some third party plugins. The interface has been cleaned compared to a standard QGIS installation, it shows only relevant buttons for model building and analysing. 
+
 Since March 4th 2019, the Modeller Interface is available for QGIS stable releases (3.4 and 3.10). For 3Di results before this date an older version of the plugin will remain available for QGIS 2.18. 
```

Comment:
Bij de werkende versie is het mij niet helemaal duidelijk. Ze zijn niet consequent met wat er op github staat. Daarnaast heb ik gevonden dat de 3Di toolbox niet werkt in qgis 3.10.12. Even kijken wat hier moet staan.